src/scenario-guide/azext_scenario_guide/_help.py

```diff
@@ -0,0 +1,51 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['scenario'] = """
+type: group
+short-summary: E2E Scenario Usage Guidance
+"""
+
+
+helps['scenario guide'] = """
+type: command
+short-summary: Fuzzy search E2E scenario examples and execute them with walk-through
+long-summary: >
+
+    This an intelligent guidance tool that can help you find and execute E2E scenario samples!
+    It supports fuzzy search and sort the searched results according to the matching degree from high to low.
+    It supports flexible search scenarios, you can specify the search scope and customize the matching rules.
+    Moreover, it helps you execute these E2E scenarios more efficiently with a friendly walk-through process.\n
+
+    There are some custom configurations:\n
```

Comment:
@chasewilson Please help review these help descriptions when you are free, thanks!

Reply:
Adding @dbradish-microsoft for doc review.